Orlando Walking Tours/Controllers/LocationListVC.swift

```diff
@@ -195,6 +203,29 @@ class LocationListVC: UIViewController
     }
 
     ////////////////////////////////////////////////////////////
+
+    @IBAction func sortButtonTapped(_ sender: Any)
+    {
+        if self.sortingOrder == .byLocation
+        {
+            self.sortAlphabetically()
+            self.sortButton.setTitle("Current Location", for: .normal)
+            self.sortingOrder = .alphabetical
+        }
+        else
+        {
+            self.sortClosestToMe()
+            self.sortButton.setTitle("Location Name", for: .normal)
+            self.sortingOrder = .byLocation
+        }
+
+        DispatchQueue.main.async
```

Comment:
Yeah, I was having issues getting the alphabetically sorted array to display in the table view...  I mistakenly thought it was because I needed to do something on the main queue but forgot to remove this code.